subprojects/language-scala/src/test/groovy/org/gradle/api/internal/tasks/scala/ZincScalaCompilerArgumentsGeneratorTest.groovy

```diff
@@ -107,4 +107,15 @@ class ZincScalaCompilerArgumentsGeneratorTest extends Specification {
         args.contains("-other")
         args.contains("value")
     }
+
+    def "adds compiler plugins"() {
+        spec.scalaCompilerPlugins = [new File("path/to/plugin1"), new File("/abspath/to/plugin2")]
+
+        when:
+        def args = generator.generate(spec)
+
+        then:
+        args.contains("-Xplugin:path/to/plugin1")
```

Comment:
This and the next assertion fail on Windows, you will have to use the `File.separatorChar` or some other method to be properly platform independent.